src/transformers/training_args.py

```diff
@@ -1335,6 +1335,17 @@ def _setup_devices(self) -> "torch.device":
                             "Looks like distributed multinode run but MASTER_ADDR env not set, "
                             "please try exporting rank 0's hostname as MASTER_ADDR"
                         )
+                if torch.get_num_threads() == 1 and get_int_from_env(["OMP_NUM_THREADS", "MKL_NUM_THREADS"], 0) == 0:
+                    import psutil
```

Comment:
`psutil` is not a hard dependency of Transformers, so you should add a `is_psutil_available` to the test above.